package.json

```diff
@@ -71,7 +72,8 @@
     "node-fetch": "2.x.x",
     "strip-ansi": "^5.2.0",
     "text-table": "^0.2.0",
-    "typescript-json-schema": "^0.38.0"
+    "typescript-json-schema": "^0.38.0",
+    "unfetch": "^4.1.0"
```

Comment:
We've been using node-fetch which uses node Request and other core stuff internally, meaning we can use tools like Polly.js for testing. Would you mind switching to that?

Reply:
Sure, good idea!